docs/dev/machine-output.adoc

```diff
@@ -1,10 +1,173 @@
-# Machine Readable Output
+= Machine readable output
 
 This document outlines all the machine readable output options and examples.
 
-`odo app describe -o json | jq`
- 
-```json
+See the below table for a list of all possible machine readable output commands:
+
+|===
+| Command | Subcommand | Implemented?
+
+| application
+| delete
+| N
+
+|
+| <<application-describe,describe>>
+| Y
+
+|
+| <<application-list,list>>
+| Y
+
+| catalog
+| describe
+| N
+
+|
+| search
+| N
+
+|
+| <<catalog-list-components,list components>>
```

Comment:
This link and the ones on lines 35, 39, 111, 115, 131, 139 do not work. Most probably cause they are not  listed below, are we planning to add those sections here? Is that why we have these links?

Reply:
Yes, that's intentional, just wanted to see if this format is fine :+1: